lib/decidim/file_authorization_handler/version.rb

```diff
@@ -2,7 +2,12 @@
 
 module Decidim
   module FileAuthorizationHandler
-    # Uses the latest matching Decidim version
-    VERSION = "0.15.0"
+    def self.version
+      "0.1.0"
```

Comment:
we can not go backwards when versioning... We moved the definition of the version, but the version in the gemspec should be coherent.

let's keep with the `# Uses the latest matching Decidim minor version` comment and upgrade to v `0.25.0`

Reply:
oh! ok, not problem :)